frontend/src/api/auth.js

```diff
@@ -0,0 +1,29 @@
+import fetcher from './helpers';
+
+const baseUrl = '/api/auth';
+
+const loginUser = ({ email, password }) =>
+  fetcher(`${baseUrl}/login`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+    body: JSON.stringify({
+      email,
+      password,
+    }),
+  });
+
+const registerUser = ({ email, password }) =>
+  fetcher(`${baseUrl}/register`, {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
```

Comment:
Can this be also set as a default in the `fetcher` since probably all of the content being sent will be `JSON`?

Reply:
that is what I meant with my first comment too. I'll try to be more descriptive next time 😅